src/clients/bridges/CrossChainTransferClient.ts

```diff
@@ -13,37 +13,39 @@ export class CrossChainTransferClient {
 
   // Get any funds currently in the canonical bridge.
   getOutstandingCrossChainTransferAmount(
-    address: string,
+    address: Address,
     chainId: number | string,
```

Comment:
Driveby: `number` seems to be fine here, and then we can drop the `Number()` conversion on line 22 below.

```suggestion
    chainId: number,
```

Reply:
Implemented (as with the comments below) https://github.com/across-protocol/relayer/pull/2328/commits/98dec345837e7dac9b2dc2721976f608c169c2ab